.github/labeler-config.yml

```
@@ -1,9 +1,9 @@
-# Add 'config-change' label to any file changes within the 'docs/config' folder or the docs/README.md file
+# Add 'config-change' label to any file changes related to configuration
 config-change:
 - changed-files:
```

YAMLlint (1.37.1)

[error] wrong indentation: expected at least 1

(indentation)